src/content/en/2022/javascript.md

```diff
@@ -16,7 +16,864 @@ featured_stat_2: TODO
+
+React usage notably remained the same from last year at 8%, which may be a signal that adoption of the library has plateaued due to an increasing amount of choices in the JavaScript ecosystem.
+
+#### Libraries used together
+
+It's not an uncommon scenario to see multiple frameworks and libraries used on the same page. As with last year, we'll examine this phenomenon to gain insight into how many libraries and frameworks have been used together in 2022.
+
+<figure markdown>
+| Libraries                                              | Desktop  | Mobile  |
+| ------------------------------------------------------ | -------- | ------- |
+| jQuery                                                 | 10.19%   | 10.33%  |
+| jQuery, jQuery Migrate                                 | 4.30%    | 4.94%   |
+| core-js, jQuery, jQuery Migrate                        | 2.48%    | 2.80%   |
+| core-js, jQuery                                        | 2.78%    | 2.74%   |
+| jQuery, jQuery UI                                      | 2.40%    | 2.07%   |
+| core-js, jQuery, jQuery Migrate, jQuery UI             | 1.18%    | 1.36%   |
+| jQuery, jQuery Migrate, jQuery UI                      | 0.88%    | 0.99%   |
+| GSAP, Lodash, Polyfill, React                          | 0.48%    | 0.93%   |
+| Modernizr, jQuery                                      | 0.87%    | 0.86%   |
+| core-js                                                | 0.92%    | 0.85%   |
+
+  <figcaption>{{ figure_link(
+    caption="Analysis of libraries and frameworks used together on observed pages.",
+    sheets_gid="1200090523",
+    sql_file="frameworks_libraries_combos.sql"
+  ) }}</figcaption>
+</figure>
+
+It's clear though that jQuery has some serious staying power, with some combination of it, its UI framework, and its migration plugin occurring in the top seven spots, with core-js having a prominent role in library usage as well.
+
+#### Security vulnerabilities
+
+Given the wide proliferation of JavaScript on today's web, and with the advent of installable JavaScript packages, it's no surprise that security vulnerabilities exist in the JavaScript ecosystem.
+
+{{ figure_markup(
+    caption="The percentage of mobile pages that download a vulnerable JavaScript library or framework.",
+    content="57%",
+    classes="big-number",
+    sheets_gid="1881889053",
+    sql_file="lighthouse_vulnerabilities.sql"
+  )
+}}
+
+While 57% of mobile pages serving up a vulnerable JavaScript library or framework is significant, this figure _is_ down from last year's figure of 64%. This is encouraging, but there's quite a bit of work to be done to lower this figure. We hope that as more security vulnerabilities are patched, developers will be incentivized to update their dependencies to avoid exposing their users to harm.
```

Comment:
```suggestion
While 57% of mobile pages serving up a vulnerable JavaScript library or framework is significant, this figure is down from last year's figure of 64%. This is encouraging, but there's quite a bit of work to be done to lower this figure. We hope that as more security vulnerabilities are patched, developers will be incentivized to update their dependencies to avoid exposing their users to harm.
```